# Final-Project-Brewer_DS
This is the final project submission by Dejah Smith for the Practical Computing course.

My project is an extension of an ongoing initiative that is measuring the affects of human activities related to land use on nutrient cycles across a low-nutrient coastal plain wetland. The structure of the experiment involves 8 plots that undergo either a mowed treatment (M) or a mowed fertilized treatment (MF). The results of this ecological experiement were the foundation to my research. Sanger sequencing was conducted on 10 soil bacterial isolate pairs from the different plot treatments. I conducted whole genome sequencing to identify metabolic and physiological differences between the pairs.

For my submission, I chose to highlight two bacterial isolate species: Arthrobacter ramosus and Pseudomonas koreensis 
I used a pangenomic workflow to analyze the pairs and the presence/absence of genes from mowed (M) and mowed fertilized (MF) plots.
Pangeomic analyses allow you to view a full set of genes and determine their homology as well as the functions they share. This is important to know for my research because I am analyzing not only the genes present across the two treatments but how they may overlap between microbial species and the influence these genes have on plant-microbe interactions.

Arthrobacter species are known to be oligotrophs, meaning they can live in environments with low nutrients. Pseudomonas species are known to be copiotrophs, meaning they live in environments with high levels of nutrients, thriving off of Carbon availability.

Using a Pangenomic workflow, I have analyzed these two bacterial species and provided some early conclusions to the results.
